packages/ublue-os-update-reminder/src/usr/lib/systemd/user/ublue-reboot-notifier.service

```
@@ -0,0 +1,15 @@
+[Unit]
+Description=Monitor screen unlock and show reboot notifications
+After=graphical-session.target
+ConditionPathExists=/run/user/%i/ublue-reboot-notifier-enabled
+
+[Service]
+Type=simple
+ExecStart=/usr/libexec/ublue-reboot-notifier-watcher.py
+Restart=on-failure
+RestartSec=5
+StartLimitBurst=3
+StartLimitIntervalSec=600
```

```suggestion
[Unit]
Description=Monitor screen unlock and show reboot notifications
After=graphical-session.target
ConditionPathExists=/run/user/%i/ublue-reboot-notifier-enabled
StartLimitIntervalSec=600

[Service]
Type=simple
ExecStart=/usr/libexec/ublue-reboot-notifier-watcher.py
Restart=on-failure
RestartSec=5
StartLimitBurst=3
```
I get these error, `StartLimitIntervalSec=600` should be in `[Unit]` it seems
```
/usr/lib/systemd/user/ublue-reboot-notifier.service:12: Unknown key 'StartLimitIntervalSec' in section [Service], ignoring.
```